### DISCUSSION TITLE
groups and boxes

### DISCUSSION THREAD

**Discussion by WvBra** [sandboxie-plus/Sandboxie#2704] [Q&A Feedback] [answered]
If a group is created in sandboxie make a directory where all the members (sandboxes) are stored in. So default groep is on disk ..username\sandbox\default\boxname 1 etc. Groupname edge-procject will create a directory on disk where the members of this group are stored. So boxname 1 in egde-project is another than boxname 1 in default. Why? It makes easier to create backups and manage all the boxes.

**Comment by bastik-1001** [comment]
Is this suggesting that the boxname isn't supposed to be unique anymore?

What is supposed to be happening when a box is moved out of group, maybe even into another? Then the whole structure would need to move.

**Comment by WvBra** [comment]
If you have something like "grouppath" voor eacht group, then the only thing that has to be done is combine grouppath and structure. The advantage of this method is that one can create separate maintenance of backup jobs for each group as a whole and still have the possiblility to backup one or more boxes seperate.

I have 3 groups. In the first group i create boxes that i use to seperate webbrowsing for each client and store documents for those clients. The second group is used for projects for wihich i collect information that i want to be seperated from other projects.

The third group consist of boxes that are cleared completly after exit last running program. I use these boxes mostly for tricky searches on internet. This group is never input for backup.



----- Original Message -----

**Reply by bastik-1001** [reply]
I see the point of that use-case. Thank you for elaborating.

**Comment by offhub** [comment, marked as answer]
You can achieve this by defining a separate `FileRootPath` for each box. 

```
FileRootPath=\??\%SystemDrive%\Sandbox\%USER%\webbrowsing\%SANDBOX%
.
FileRootPath=\??\%SystemDrive%\Sandbox\%USER%\projects\%SANDBOX%
.
FileRootPath=\??\%SystemDrive%\Sandbox\%USER%\temp\%SANDBOX%
```

https://github.com/sandboxie-plus/Sandboxie/issues/2709#issuecomment-1435927408
